### ISSUE TITLE
Unable to switch terminal to normal mode

### ISSUE THREAD

**Issue by baldwindavid** [onivim/oni2#2530]
I fear there is something obvious that I am doing wrong here, but I can't seem to switch from Terminal "insert" to "normal" mode. I have tried `ctrl + \`, `ctrl + n` as noted at https://onivim.github.io/docs/using-onivim/integrated-terminal#normal-mode

Also tried the same with `cmd` and with a capital `N`. I'm on a Mac (10.15.6). I notice in your screenshare that your key displayer is showing `Ctrl + N` while mine displays either `Ctrl + n` or `Ctrl + Shift + n` when trying lowercase or uppercase `n`.

**Comment by CrossR** [comment]
You aren't doing anything wrong! There was a bug blocking this which has been fixed now, but it looks like the Early Access portal is 2 days behind and is missing this https://github.com/onivim/oni2/pull/2516 fix.

I've dropped @bryphe a message to poke the machine, so hopefully it should catch up in a few hours. Very occasionally it will get stuck and we fix and issue on the GitHub side, but not on the internal CI side.
